modules/setting/setting.go

```diff
@@ -612,6 +612,7 @@ func loadFromConf(allowEmpty bool, extraConfig string) {
 
 	Cfg.NameMapper = ini.SnackCase
 
+	// TODO: when running gitea as a sub command inside git, the HOME directory is not the user's home directory
```

Comment:
homeDir appears to be only used to set the default SSH.RootPath and isn't used anywhere else. Perhaps we should move this closer to its use

Reply:
```suggestion
```

OK, I delete this comment here. If there is new suitable place to write this comment, let me know (or edit on this PR directly).